modules/reindex/src/main/java/org/elasticsearch/index/reindex/remote/RemoteInfo.java

```diff
@@ -24,32 +24,45 @@
 import org.elasticsearch.common.io.stream.StreamInput;
 import org.elasticsearch.common.io.stream.StreamOutput;
 import org.elasticsearch.common.io.stream.Writeable;
+import org.elasticsearch.common.unit.TimeValue;
+import org.elasticsearch.index.reindex.BulkByScrollTask;
 
 import java.io.IOException;
 import java.util.HashMap;
 import java.util.Map;
 
 import static java.util.Collections.unmodifiableMap;
 import static java.util.Objects.requireNonNull;
+import static org.elasticsearch.common.unit.TimeValue.timeValueMinutes;
 
 public class RemoteInfo implements Writeable {
+    /**
+     * Default socket timeout for requests that don't have one set.
+     */
+    public static final TimeValue DEAULT_SOCKET_TIMEOUT = timeValueMinutes(1);
```

Comment:
Thanks for finding the typo! I'll certainly fix it.

Do you think I should drop the default to 30 seconds so it lines up with the transport client? Those are different timeouts (connect vs socket) but they are similar enough. Do you think I should change the rest client's default rather than have a default for reindex at all?